controllers/usersController.js

```diff
@@ -48,6 +48,25 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Checks whether a user exists or not
```

Comment:
Should be something like `checks whether a given username is available`

Reply:
done